### PR TITLE
[KOGITO-850] Add Cucumber test for Prometheus integration

### PR DESCRIPTION
**Pull request by sutaakar**
https://issues.redhat.com/browse/KOGITO-850

Signed-off-by: Karel Suta <ksuta@redhat.com>

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: test/smoke/openshift.go [hunk 1]
```diff
@@ -133,3 +156,34 @@ func waitAndRetrieveRouteURI(namespace, serviceName string) (string, error) {
 	GetLogger(namespace).Debugf("Got route %s\n", routeURI)
 	return routeURI, nil
 }
+
+// waitForPods waits for pods with specific label to be available and running
+func waitForPods(namespace, labelName, labelValue string, numberOfPods, timeoutInMin int) error {
```
Comment (ricardozanini) [inline test/smoke/openshift.go]: Since those functions are related to `Pods` resource, why not keep them on `kubernetes.go` instead?

Comment (sutaakar) [reply test/smoke/openshift.go]: good point, can do that in next PR, can move there also route functions

**Comment by sutaakar** [comment]
@ricardozanini Good point, added JIRA for this test.

File: test/smoke/openshift.go (hunk 1 above)
Comment (sutaakar) [reply test/smoke/openshift.go]: Fixed in this PR.

**Comment by sutaakar** [comment]
@ricardozanini Can you please take a look again?

**Pull request merged**